### PR TITLE
[DDS-625] Sumologic Log Transport

### PR DESCRIPTION
**Pull request by FleetAdmiralButter**
## Motivation and Context

**JIRA issue:** https://digital-vic.atlassian.net/browse/DDS-625

## Changed

1. Implemented the Winston UDP log forwarding to send logs to the Sumo Logic OTel collector.
2. Removed the Logstash transport.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.

### CODE REVIEW

**Comment by waitingallday** [comment]
@FleetAdmiralButter we use yarn for ripple

File: packages/ripple-nuxt-tide/lib/core/logger.js [hunk 1]
```diff
@@ -83,29 +83,25 @@ let logger = createLogger({
 
 // Use Logstash transport in Lagoon server instead of console
 if (process.env.LAGOON_GIT_SAFE_BRANCH && !process.client) {
-  // Create the Lagoon Logstash transport.
-  // Add lagoon required meta.
-  const lagoonFormat = format(info => {
-    const LAGOON_LOGS_DEFAULT_SAFE_BRANCH = 'safe_branch_unset'
-    const LAGOON_LOGS_DEFAULT_LAGOON_PROJECT = 'project_unset'
-    const k8sNamespace = process.env.LAGOON_PROJECT || LAGOON_LOGS_DEFAULT_LAGOON_PROJECT
```
Comment (nicksantamaria) [inline packages/ripple-nuxt-tide/lib/core/logger.js:91]: Are we losing the namespace in the payload?

Comment (nicksantamaria) [reply packages/ripple-nuxt-tide/lib/core/logger.js:91]: Answering my own question - no. It is included as `source_host`

**Review by nicksantamaria — APPROVED** [review]
My approval is merely of the payload. I'll leave analysis of the JS implementation to one of the frontend devs.

**Review by dylankelly — APPROVED** [review]
LGTM 👍 
@nicksantamaria can you confirm this is tested and if we can merge this?

**Pull request merged**